creyes17/affirm-cares-github-workshop
# Updated slides with my favorite hobby

Edited a slide to include information about my favorite hobby.

To test:
- Open index.html and see that the slides still load.
- Go to my slide and see that you can see the information.
- Go the the next slide and see that it still loads correctly
